hips/tiles/tile.py

```diff
@@ -0,0 +1,131 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.utils.data import get_pkg_data_filename
```

Comment:
I think this `from astropy.utils.data import get_pkg_data_filename` can be removed from this file, it isn't used, no?